lib/Doctrine/ORM/Mapping/AssociationOverrides.php

```diff
@@ -9,13 +9,7 @@
 use function array_values;
 use function is_array;
 
-/**
- * This annotation is used to override association mappings of relationship properties.
- *
- * @Annotation
- * @NamedArgumentConstructor()
- * @Target("CLASS")
- */
+/** This attribute is used to override association mappings of relationship properties. */
```

Comment:
You've rephrased the documentation on each attribute class to use the term "attribute" instead of "annotation". Would it make sense to do this on the 2.14.x branch already?